### DISCUSSION TITLE
Update to know that CJS is the gitflow

### DISCUSSION THREAD

**Discussion by bobstuart** [CJ-Systems/gitflow-cjs#42] [Ideas]
Right now I "think" I've installed the CJS version of git flow but the version info from a repo says:
git flow version
2.0.0 (AVH Edition)

I did update the installer shell script to have the CJS repo instead of the AVH one as default. That's another thing that would be nice to have changed. 

Is there something other than the git flow version I should look for to ensure I have the right git flow installed?

**Comment by ChrisJStone** [comment]
> Right now I "think" I've installed the CJS version of git flow but the version info from a repo says: git flow version 2.0.0 (AVH Edition)
> 
> I did update the installer shell script to have the CJS repo instead of the AVH one as default. That's another thing that would be nice to have changed.
> 
Thank you for catching that. I will git a hotfix pushed out addressing the incorrect naming of the edition along with the correcting the repo in the installer scripts. 


> Is there something other than the git flow version I should look for to ensure I have the right git flow installed?

If you still have the gitflow-cjs repo on your system you can look at the README.md file. The first line should read git-flow (CJS Edition) also there should be two changelog files present one titled CHANGELOG-AVH.md which is the old changelog and the now current one CHANGELOG.md which will have only the 2.0.0 release in it. As a final check if you do not have the repo on your system you can in a git flow enabled repo run the command git flow release start --help. If you do not see the following in the help output then you have git-flow-cjs installed

-v, --verbose         Verbose (more) output

**Reply by ChrisJStone** [reply]
Hotfix release is now live.
